### DISCUSSION TITLE
no tsconfig found with .vue files Vscode

### DISCUSSION THREAD

**Discussion by DataHearth** [vuejs/language-tools#1345] [Q&A]
Hello there!

I'm experiencing the weirdest behavior with vscode. My vscode doesn't detect a tsconfig from certain `.vue` files and with a certain typo case. 

Here's my example:
```
.
├── app.vue
├── components
│  └── header.vue
├── lib
│  ├── rules.ts
│  └── schemas.ts
├── node_modules
├── nuxt.config.ts
├── package.json
├── pages
│  ├── index.vue
│  └── login.vue
├── plugins
│  ├── firebase.client.ts
│  └── naive.ts
├── pnpm-lock.yaml
├── public
│  └── favicon.ico
├── README.md
└── tsconfig.json
```

As you can see, I have 2 pages. One named `index.vue` and another one `login.vue`. With the first one, vscode detect a tsconfig file, but for the second one, it does not. BUT, if I rename `login.vue` to `Login.vue`, it kinda works (my code flashes like garland but it works). Same behavior with my component `header.vue` but reversed (uppercase no, no case, yes).

Everything is working properly when building

Does anyone facing this weird and tilting issue ?

`tsconfig.json`:
```
{
  // https://v3.nuxtjs.org/concepts/typescript
  "extends": "./.nuxt/tsconfig.json"
}
```

screenshot:
<img width="615" alt="image" src="https://user-images.githubusercontent.com/28595242/170079731-f9f89003-4aba-4ae0-87b1-8c511ede8886.png">

INFO:
OS: darwin
Nuxt: v3.0.0.rc3
package manager: pnpm
vscode: 1.67.2
Volar: 23/05/2022, 03:48:53

**Comment by johnsoncodehk** [comment]
Sorry for late, can you reproduce this problem in v0.40.13 or v1.0.0-alpha.x?

**Reply by DataHearth** [reply]
Hey, if I remember correctly, the problem was the naming of `header.vue`. I managed to reproduce it with a barebone vue project. 
It seems the extension (or something else related to vue) doesn't like at all having a file named with either `Header` or `header`.

**Reply by johnsoncodehk** [reply]
It maybe fixed in https://github.com/johnsoncodehk/volar/commit/302b5733639f8576ac6423bec7912c47a50465a9
